resources/assets/architects_palette/lang/en_us.json

```diff
@@ -226,6 +230,7 @@
   "block.architects_palette.withered_osseous_brick_vertical_slab": "Withered Osseous Brick Vertical Slab",
   "block.architects_palette.wither_lamp": "Wither Lamp",
   "item.architects_palette.withered_bone": "Withered Bone",
+  "tag.c.withered_bones": "Withered Bones",
```

Comment:
What's with the c here?

Reply:
"c" is the "common" tag in Fabric. This is equivalent to "forge:bones/wither" in the Forge project.

Reply:
Oh, alright. With that, I'll go ahead and merge this.